### PR TITLE
Default state present in container-infra.yml

### PR DESCRIPTION
**Pull request by brtknr**

### CODE REVIEW

File: config/k8s-demo.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ lln_name: p3-lln
 # SIP configuration
 cluster_user: "fedora"
 cluster_name: "k8s-demo"
-cluster_template_name: "k8s-fedora-atomic-27"
+cluster_template_name: "k8s-fedora-atomic-29"
```
Comment (dougszumski) [inline config/k8s-demo.yml]: I guess you are only upgrading the k8s demo playbook and nothing else using atomic (`config/kubernetes.yml`, `config/swarm-sip.yml`)? A hint in the commit message would be helpful :)

File: config/openhpc.yml [hunk 2]
```diff
@@ -50,13 +50,6 @@ slurm_compute:
   num_nodes: "{{ slurm_compute_num_nodes }}"
   user: "centos"
 
-slurm_computex:
```
Comment (dougszumski) [inline config/openhpc.yml]: This has been removed because it is not used anywhere?

File: ansible/container-infra.yml [hunk 3]
```diff
@@ -10,7 +10,7 @@
     os_container_infra_auth: "{{ openstack_auth }}"
     os_container_infra_auth_type: "{{ openstack_auth_type }}"
     os_container_infra_cluster_name: "{{ cluster_name }}"
-    os_container_infra_state: "{{ cluster_state | default('query') }}"
+    os_container_infra_state: "{{ cluster_state | default('present') }}"
```
Comment (dougszumski) [inline ansible/container-infra.yml]: As someone not familiar with this, it would be helpful to explain why this change has been made in the commit message.

File: config/openhpc.yml (hunk 2 above)
Comment (brtknr) [reply config/openhpc.yml]: Yep

Comment (brtknr) [reply config/openhpc.yml]: I added `slurm_computex` as a test and managed to accidentally commit it.

File: ansible/container-infra.yml (hunk 3 above)
Comment (brtknr) [reply ansible/container-infra.yml]: This allows the behaviour to match with that of cluster-infra.
Ensuring state is present is a more common use case than query.

Comment (brtknr) [reply ansible/container-infra.yml]: I have expanded the commit message with this info to that end.

**Comment by brtknr** [comment]
Thanks
